node/src/stream_handler_pool.rs

```diff
@@ -155,6 +157,17 @@ impl Handler<DispatcherNodeQueryResponse> for StreamHandlerPool {
     }
 }
 
```

Comment:
This thing below is all good. My only request is, if you haven't done yet, please create a card with tiny description about implementing this through a `derive` macro.

Also, I'd prefer to have a simple TODO here referring to the newly existing card.

Reply:
I've created the card, and marked a TODO referencing it - https://github.com/MASQ-Project/Node-issues/issues/686